lua/refactoring/tests/refactor_spec.lua

```diff
@@ -51,6 +51,8 @@ local function for_each_file(cb)
 end
 
 local function test_empty_input()
+    -- TODO (TheLeoP): does this make sense?
```

Comment:
I don't think so - could you elaborate as to what was the intention here?

Reply:
This particular test checks if an error message is the correct one by calling the refactoring function with `pcall`. Since in this PR in using  `vim.notify` to show this errors, in order for  `pcall` to be able to intercept them I overwrote `vim.notify` with `error` for this particular test.

Reply:
Sounds good, I think that makes sense in that case; tests can't pass without it either.